### DISCUSSION TITLE
Multipart workflow design

### DISCUSSION THREAD

**Discussion by Varamil** [dune3d/dune3d#145] [General]
I've started to use dune3D, and the tool is very promising, but I'm wondering what is the workflow that should be followed to design an object with multiple elements, and with some of them duplicated.

I see different options, using bodies to separate parts or using document links, but I don't figure out which is the best solution.
With bodies, I don't see how to reuse a part several times (with the feature to modify one to update all). And with the document link, it seems that we can only constrain the document origin, which looks very limited to make an assembly.

 I don't know if I'm missing something or if some features need to be developed, but what are your suggestions to design parts and make a final assembly of them?

Thanks

**Comment by carrotIndustries** [comment]
> And with the document link, it seems that we can only constrain the document origin, which looks very limited to make an assembly.

Yeah, when implementing document entities, I initially had in mind to add anchors similar to clusters that can be used to reference document entities. However, after having implemented them, I ran out of motivation to add them.

Now that people actually need document anchors, I might implement them in the future.
